punjab/session.py

```diff
@@ -379,11 +379,17 @@ def _pollForId(self, d):
 
 
     def connectEvent(self, xs):
-
         self.version =  self.authenticator.version
         self.xmlstream = xs
         self.xmlstream.rawDataOutFn = self.rawDataOut
         self.xmlstream.rawDataInFn = self.rawDataIn
+        if self.pint.proxy_protocol:
```

Comment:
Then why did you enable proxy_protocol flag? in that case all your request fail as the backend well get garbage.